app/dao/api_key_dao.py

```diff
@@ -10,6 +12,26 @@
 from app.models import ApiKey
 
 
+@transactional
+def resign_api_keys(unsafe: bool = False):
+    # Resign the secret column of the api_keys table
+    # This allows us to rotate the secret key used to sign the api key secret
```

Comment:
Let's write a proper python doc string for the function and we can document the parameter as well, i.e. what it does.